app/src/androidTest/java/com/android/unio/components/event/EventDetailsTest.kt

```diff
@@ -81,8 +82,16 @@ class EventDetailsTest : TearDown() {
     MockKAnnotations.init(this, relaxed = true)
     eventPictures =
         listOf(
-            EventUserPicture("12", "http:image.com", User.emptyFirestoreReferenceElement(), 0),
-            EventUserPicture("34", "http:image.com", User.emptyFirestoreReferenceElement(), 3))
+            EventUserPicture(
+                "12",
+                "https://i1.sndcdn.com/artworks-000055240636-kowvdx-t500x500.jpg",
```

Comment:
What happend if the URL is not available (like a 404 error) ? If it does not make the tests crash, you can leave it as this, but otherwise consider downloading the image or use one already downloaded

Reply:
Yes you are right, it made the test fail. I found a way to get the local Uri of a drawable instead.